.github/workflows/cijoe_packages.yml

```diff
@@ -52,7 +52,15 @@ jobs:
 
     - name: Run it!
       run: |
-        cijoe --monitor -l
+        coverage run\
+          --data-file=cijoe-output/.coverage\
+          --rcfile=repository/.coveragerc\
+          -L \
+          $(which cijoe) --monitor
+        coverage\
```

Comment:
How about moving the "coverage report" down to the collection-stage? Then there is a stage which run cijoe and produces the cijoe-report.